src/applications/vaos/services/appointment/transformers.vaos.js

```diff
@@ -0,0 +1,63 @@
+import { TYPES_OF_CARE, APPOINTMENT_TYPES } from '../../utils/constants';
+
+function getAppointmentType(appt) {
+  if (appt.kind === 'cc' && appt.start) {
+    return APPOINTMENT_TYPES.ccAppointment;
+  } else if (appt.kind === 'cc' && appt.requestedPeriods?.length) {
+    return APPOINTMENT_TYPES.ccRequest;
+  } else if (appt.kind !== 'cc' && appt.requestedPeriods?.length) {
+    return APPOINTMENT_TYPES.request;
+  }
+
+  return APPOINTMENT_TYPES.vaAppointment;
+}
+
+export function transformVAOSAppointment(appt) {
```

Comment:
Transformer for the returned request data. Should be pretty close to the full transform for appointments from the VAOS service, but with some unmapped data that we'll do in later tickets (primarily video).